todocore/src/main/java/com/knighttodo/todocore/rest/DayResource.java

```diff
@@ -147,4 +141,25 @@ public void deleteDay(@PathVariable UUID dayId) {
             throw new DayCanNotBeDeletedException("Day can't be deleted.", ex);
         }
     }
+
+    @GetMapping("/date")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Find all Days by Date", response = DayResponseDto.class, responseContainer = "List")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Days found"),
+            @ApiResponse(code = 400, message = "Invalid operation"),
+            @ApiResponse(code = 403, message = "Operation forbidden"),
+            @ApiResponse(code = 500, message = "Unexpected error")
+    })
+    public List<DayResponseDto> findDaysByDate(@RequestParam(name = "date") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) Date date) {
+        try {
+            return dayService.findDayByDate(date)
+                    .stream()
+                    .map(dayRestMapper::toDayResponseDto)
+                    .collect(Collectors.toList());
```

Comment:
let's do it in Service, we should not have any mappers in Resources (Controllers)

Reply:
I deleted list and return only one day.